cloud/ec2.go

```diff
@@ -50,7 +50,7 @@ type EC2ProviderSettings struct {
 	KeyName string `mapstructure:"key_name" json:"key_name,omitempty" bson:"key_name,omitempty"`
 
 	// MountPoints are the disk mount points for EBS volumes.
-	MountPoints []MountPoint `mapstructure:"mount_points" json:"mount_points,omitempty" bson:"mount_points,omitempty"`
+	MountPoints []MountPoint `mapstructure:"mount_points" json:"mount_points" bson:"mount_points,omitempty"`
```

Comment:
I think we should still keep it for slices, I made [this example playground](https://go.dev/play/p/lLx_uzHRq3k) which shows they do get filtered when the slice is empty

Reply:
Thanks